### ISSUE TITLE
The requested module does not provide an export named 'default'

### ISSUE THREAD

**Issue by tolgahanbeyazoglu** [egidiusmengelberg/nuxt-aos#25]
I installed nuxt js aos to get quick response but I get this error how can I make it disappear quick solution
![Ekran görüntüsü 2024-03-31 181653](https://github.com/egidiusmengelberg/nuxt-aos/assets/129540736/fe0f778b-3352-485b-9a41-83b3c46b2c28)

**Comment by egidiusmengelberg** [comment]
what version of nuxt-aos are you using?

**Comment by tolgahanbeyazoglu** [comment]
@egidiusmengelberg  problem solved I updated nuxt js to the latest version

**Comment by vinumweb** [comment]
I still get this problem on the latest version of Nuxt... Any fixes?

**Comment by egidiusmengelberg** [comment]
What version of nuxt-aos do you have installed?

**Comment by vinumweb** [comment]
@egidiusmengelberg I installed the latest version from pnpm with the installation command provided in the docs.

**Comment by egidiusmengelberg** [comment]
Thats weird. I'll investigate this week. Maybe something specific when using pnpm.

**Comment by jaybharadia** [comment]
This Issue happens in below version

"nuxt": "^3.13.0",
"nuxt-aos": "^1.2.5"

**Comment by martijn-b3** [comment]
i also face the same issue, i have:
    "nuxt": "^3.13.2",
    "nuxt-aos": "^1.2.5",

Using pnpm

**Comment by egidiusmengelberg** [comment]
yeah I know, unfortunately I do not have the time to investigate. I would use a nuxt plugin to enable aos in your app.

[https://stackoverflow.com/questions/71484930/using-aos-with-nuxt3](https://stackoverflow.com/questions/71484930/using-aos-with-nuxt3
)

**Comment by tilenpirih** [comment]
Have the same issue with pnpm (bun was working fine). I solved this issue by adding aos
```pnpm add aos```
and in nuxt config add
```
vite: {
  optimizeDeps: {
    include: ['aos']
  },
}
```

although atleast for me something is still a bit buggy with SSR. Not sure it this was also before but I have huge problems with layout shift

**Comment by simonmaass** [comment]
i have the same problem with nuxt 4

**Comment by dethdkn** [comment]
This is a nuxt `4.2.0` error only, downgrading to `4.1.3` solved for me

**Comment by simonmaass** [comment]
Nuxt 4.2 tightened its Vite integration and ESM/CJS interop. That change surfaces a long-standing quirk in nuxt-aos: it imports aos as if it had a true ESM default export. In 4.2, the (updated) Vite/SSR pipeline no longer auto-unwraps CommonJS defaults the same way
